packages/core/src/render3/styling_next/bindings.ts

```diff
@@ -404,18 +404,22 @@ export function flushStyling(
     if (!isContextLocked(stylesContext, hostBindingsMode)) {
       lockAndFinalizeContext(stylesContext, hostBindingsMode);
     }
-    applyStylingViaContext(
-        stylesContext, renderer, element, data, state.stylesBitMask, setStyle, styleSanitizer,
-        hostBindingsMode);
+    if (state.stylesBitMask) {
```

Comment:
I wonder if writing it this way has a better performance?
```suggestion
    if (state.stylesBitMask !== 0) {
```